src/dictionary/manager.ts

```diff
@@ -187,9 +190,10 @@ class SchemaManager {
   loadSchemaByVersion = async (
     name: string,
     version: string,
-  ): Promise<dictionaryEntities.SchemasDictionary> => {
-    const newSchema = await dictionaryRestClient.fetchSchema(this.schemaServiceUrl, name, version);
-    return newSchema;
+  ): Promise<dictionaryEntities.SchemasDictionary | undefined> => {
+    return await dictionaryRestClient
+      .fetchSchema(this.schemaServiceUrl, name, version)
+      .catch(err => undefined);
```

Comment:
when you are using await catch is not the best to use, you should use `try / catch`, log the error and decide to return or re throw the error